parm/config/gfs/config.base

```diff
@@ -178,6 +178,7 @@ export DO_WAVE="NO"
 export DO_OCN="NO"
 export DO_ICE="NO"
 export DO_AERO="NO"
+export DO_PREP_OBS_AERO="YES"
```

Comment:
@CoryMartin-NOAA How about setting this to `NO` for now?

Reply:
Yes I think that is all that is needed to ensure it doesn't break anything